test/controllers/tasks/active.pscs.controller.unit.ts

```diff
@@ -0,0 +1,21 @@
+jest.mock("../../../src/middleware/company.authentication.middleware");
+
+import mocks from "../../mocks/all.middleware.mock";
+import { ACTIVE_PSCS_PATH } from "../../../src/types/page.urls";
+import request from "supertest";
+import app from "../../../src/app";
+import { companyAuthenticationMiddleware } from "../../../src/middleware/company.authentication.middleware";
+
+const mockCompanyAuthenticationMiddleware = companyAuthenticationMiddleware as jest.Mock;
+mockCompanyAuthenticationMiddleware.mockImplementation((req, res, next) => next());
+
+const COMPANY_NUMBER = "12345678";
+
+describe("Active pscs controller tests", () => {
+  it("should navigate to the active pscs page", async () => {
+    mocks.mockAuthenticationMiddleware.mockClear();
```

Comment:
just one more small thing, can you put the mockClear() in a beforeEach() function? that way it will be cleared when other tests get added in the future as well.

Reply:
I'd probably add it in then as beforeEach implies that there are multiple tests there.

Reply:
I'd say it is good practice to do it now as you aren't guaranteed to be the next person working on it and you might forget that it needs to be cleared between tests